<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main_window</class>
 <widget class="QMainWindow" name="main_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Set Timer App</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Downloads/stopwatch.png</normaloff>../../Downloads/stopwatch.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(212, 239, 220);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="mainwdw_lbl_showtime">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 28pt &quot;Comic Sans MS&quot;;
background-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(255, 235, 235, 206), stop:0.35 rgba(255, 188, 188, 80), stop:0.4 rgba(255, 162, 162, 80), stop:0.425 rgba(255, 132, 132, 156), stop:0.44 rgba(252, 128, 128, 80), stop:1 rgba(255, 255, 255, 0));</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="mainwdw_btn_exit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>exit.png</normaloff>exit.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="mainwdw_btn_settime">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Comic Sans MS&quot;;
background-color: rgb(180, 221, 246);
</string>
    </property>
    <property name="text">
     <string>  SET TIME</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Downloads/stopwatch.png</normaloff>../../Downloads/stopwatch.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>321</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="mainwdw_btn_play">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>play-2.png</normaloff>play-2.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mainwdw_btn_pause">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>pause-button.png</normaloff>pause-button.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mainwdw_btn_reset">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>reset.png</normaloff>reset.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
    <zorder>mainwdw_btn_pause</zorder>
    <zorder>mainwdw_btn_reset</zorder>
    <zorder>mainwdw_btn_play</zorder>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>mainwdw_btn_exit</zorder>
   <zorder>mainwdw_lbl_showtime</zorder>
   <zorder>mainwdw_btn_settime</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="mainwdw_action_about"/>
    <addaction name="mainwdw_action_main"/>
    <addaction name="separator"/>
    <addaction name="mainwdw_action_exit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="mainwdw_action_about">
   <property name="icon">
    <iconset>
     <normaloff>information.png</normaloff>information.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="mainwdw_action_exit">
   <property name="icon">
    <iconset>
     <normaloff>exit-2.png</normaloff>exit-2.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="mainwdw_action_main">
   <property name="icon">
    <iconset>
     <normaloff>stopwatch.png</normaloff>stopwatch.png</iconset>
   </property>
   <property name="text">
    <string>Timer</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
